bigchaindb/elections/election.py

```diff
@@ -35,7 +35,7 @@ class Election(Transaction):
     ELECTION_THRESHOLD = 2 / 3
 
     @classmethod
-    def get_validator_change(cls, bigchain, height=None):
+    def get_validator_change(cls, bigchain):
         """Return the latest change to the validator set
```

Comment:
It's a good time to update this docstring:

- there is no `election_id` in the collection anymore
- it does not return the latest change but the change applied at the latest block height